quesma/processors/es_to_ch_query/elasticsearch_to_clickhouse_query_processor.go

```diff
@@ -73,12 +74,14 @@ func (p *ElasticsearchToClickHouseQueryProcessor) prepareTemporaryQueryProcessor
 	logManager := clickhouse.NewEmptyLogManager(p.legacyDependencies.OldQuesmaConfig, p.legacyDependencies.ConnectionPool, p.legacyDependencies.PhoneHomeAgent(), p.legacyDependencies.TableDiscovery)
 	logManager.Start()
 
+	abTestingController := sender.NewSenderCoordinator(p.legacyDependencies.OldQuesmaConfig)
```

Comment:
Should not we create and start this `Coordinator` earlier? and pass it to `Init()`

Reply:
:+1: 
Moving to `es_to_ch_common.InitializeLegacyQuesmaDependencies`